format-delimited/src/test/java/io/cdap/plugin/format/delimited/common/DataTypeDetectorUtilsTest.java

```diff
@@ -102,7 +102,7 @@ public void testDetectDataTypeOfRowValue() {
     String[] rowValues = new String[]{"2", "24L", "3.14", "false", "100.05", "100A"};
 
     DataTypeDetectorUtils.detectDataTypeOfRowValues(new HashMap<>(), actualDataTypeDetectorStatusKeeper, columnNames,
-                                                    rowValues);
+                                                    rowValues, null);
```

Comment:
Can we use MockFailureCollector?
https://github.com/cdapio/hydrator-plugins/blob/ac2ae149e30c28a936f0860c633bba03ada634d8/core-plugins/src/test/java/io/cdap/plugin/batch/joiner/JoinerConfigTest.java#L92

Reply:
Yes, changed.